client/dashboard/task-list/index.js

```diff
@@ -256,6 +256,58 @@ class TaskDashboard extends Component {
 		);
 	}
 
+	onSkipStoreSetup = () => {
+		const completedTasksNames = this.getTasks()
+			.filter( x => x.completed )
+			.map( x => x.title );
+
+		recordEvent( 'wcadmin_tasklist_skip', {
+			completed_tasks: completedTasksNames.length,
+			completed_tasks_names: completedTasksNames,
+			skip_option: 'skip store setup'
+		} );
+
+		this.props.updateOptions( {
+			woocommerce_task_list_skip_store_setup: true,
+		} );
+	};
+
+	onDoThisLater = () => {
+		const completedTasksNames = this.getTasks()
+			.filter( x => x.completed )
+			.map( x => x.title );
+
+		recordEvent( 'wcadmin_tasklist_skip', {
+			completed_tasks: completedTasksNames.length,
+			completed_tasks_names: completedTasksNames,
+			skip_option: 'i\'ll do this later'
+		} );
+
+		this.props.updateOptions( {
+			woocommerce_task_list_do_this_later: true,
+		} );
+	};
+
+	renderSkipActions() {
+		const { skipStoreSetup, doThisLater } = this.props;
+
+		return (
+			<div style={ { textAlign: 'center' } }>
```

Comment:
We can move this into a scss file.  I think there are also some color differences between the design:

**Current**
<img width="239" alt="Screen Shot 2020-02-28 at 1 58 33 PM" src="https://user-images.githubusercontent.com/10561050/75550814-01e77780-5a33-11ea-869b-7ac4ac5d98f6.png">


**Design**
<img width="240" alt="Screen Shot 2020-02-28 at 2 00 21 PM" src="https://user-images.githubusercontent.com/10561050/75550819-0449d180-5a33-11ea-8059-42d31b983a17.png">

Reply:
Done